\documentclass{article}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}
% gensymb package, useful for general symbols, such as degrees celsius
\usepackage{gensymb}
% appropriate spaces after macro insertions
\usepackage{xspace} 

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}
\usepackage{rotating}
\usepackage[margin=2cm]{geometry}
\usepackage{color}

% enumitem allows continuous numbering across split lists.
\usepackage{enumitem}

%% EW macros
\newcommand{\mul}{\ensuremath{\mu}L\xspace}
\newcommand{\mug}{\ensuremath{\mu}g\xspace}
\newcommand{\mum}{\ensuremath{\mu}m\xspace}
\newcommand{\muM}{\ensuremath{\mu}M\xspace}
\newcommand{\degC}{\celsius\xspace}
\newcommand{\tb}[1]{\textcolor{blue}{#1}}


\begin{document}
\title{\vspace{-.75in} Anti-RNA blotting with chemiluminescent detection}
\author{Edward Wallace, ewjwallace@gmail.com}
\date{\today}
\maketitle


This protocol is for northern blotting/dot-blotting to detect specific RNAs via chemiluminescence, using biotinylated oligo-dT probes.
Gels etc are optimized for detection of 0.5-4kb mRNA transcripts.

Much of it is adapted from ULTRAhyb-Oligo and Brightstar BioDetect commercial protocols, or from Nonisotopic Probing, Blotting, and Sequencing (Kricka, ed; chapter 5).

\section*{Ingredients}

\begin{itemize}

  \item Positively Charged nylon membrane (e.g. BrightStar\texttrademark\ -Plus Positively Charged Nylon Membrane, or Roche 11-209-299-001)
\item ULTRAhyb-Oligo\textregistered\ buffer (Life technologies AM8663)
\item DNA oligo probes: 5' biotinylated, 36-45nt long, 60-75\degC hybridization temperature, 45-65\% GC-content, in 10\muM working solution. Or use biotinylated oligo-dT probes (Promega) at 5 pmol/uL.
\item Stringency wash buffer: 2X SSC (300mM NaCl, 30mM sodium citrate, pH 7.0), 0.5\% SDS. Make from 20X stock.
\item NB strip buffer: 0.1X SSC, 0.5\% SDS.
  \item BrightStar\textregistered\ BioDetect\texttrademark\ Nonisotopic Detection Kit (Life technologies AM1930), contains CDP-star, Strep-AP, wash buffer, blocking buffer, and assay buffer. Since this kit is no longer produced, follow Kricka ch. 5 and try:
  \begin{itemize}
  \item Wash buffer: 1X PBS, 0.5\% SDS. Make a 5X concentrate with 5X PBS and 2.5\% SDS.
  \item Blocking buffer: 1X PBS, 0.5\% SDS, 0.1\% I-Block reagent.
  \item Assay buffer: 0.1M DEA (diethanolamine), 1mM MgCl2, pH 10.0. % ?Make 10X concentrate.
\end{itemize}
  
\end{itemize}


\section*{Be fastidious about cleanliness and avoiding degradation!}

\begin{itemize}
\item Use RNAse-free equipment and solutions to run gel and transfer so you do not degrade your sample.
\item Only ever touch the membrane with forceps, i.e. not with your skin, sweat, etc, to avoid background.
\end{itemize}
  

\section*{Dot Blot}

To check if your probes and buffers are working without running a gel.

\begin{enumerate}
\item Blot 10uL of each RNA sample on positively charged nylon membrane under vacuum. Dry membrane.
\end{enumerate}

\section*{Northern Blotting}
\begin{enumerate}
\item Put 1uL (or other measured amount) of each RNA sample into total 10uL of 1X RNA loading buffer (NEB). Heat the tubes at 65\celsius\  for 5 min to denature RNA and then immediately place the tubes on ice. 
% For smaller RNA: 
\item Cast a 1.5\% agarose gel in 1X TBE buffer with 1:10000 SYBR safe and allow to set.
Load samples on the gel along with 5uL RNA ladder/markers. Run the gel in 1X TBE buffer at 100 V for 30min, or until the first bromophenol blue dye reaches the bottom of the gel. 
\footnote{ For smaller RNA (50-500b) use a Bio-Rad Criterion TBE-Urea 5\% acrylamide gel with 1X TBE buffer, be sure to wash wells. Run at 200 V for 90min.}
\item Turn off the current, remove the gel, and transfer the RNA onto a positively-charged nylon membrane. Use a pencil to mark the \emph{sample} side of the membrane. Assemble gel sandwich according to picture (RNA is negatively charged, so gel near negative electrode and meembrane near positive electrode), use a standard Western blotting apparatus with 0.5X TBE buffer at 1 amp for 1 h. Trim off the portions of the gel that do not contain the samples to decrease the size of the membrane and use of smaller amounts of reagents in subsequent steps.
\item Crosslink RNA to the membrane by UV at  300mJ/cm$^2$; keep the membrane wet during crosslinking. Then dry the membrane; sample side faces \emph{up} throughout.
\end{enumerate}

\section*{Hybridization}

\begin{enumerate}[resume]
\item Preheat ULTRAhyb-Oligo to 42-68\degC to dissolve all precipitated material. \label{step:prehyb}
\item Prehybridize the blot for 30 min at 42\degC. In the Drummond lab, use a yeast incubator at 42\degC, 40rpm shaking for this.
Use 1 mL ULTRAhyb-Oligo Hybridization Buffer per 10 cm$^2$ of membrane. Be certain that enough solution is present to keep the membrane uniformly wet.
\item Add 5nM of the end labeled oligonucleotide (0.5\mul of 10\muM working solution per mL of buffer). 
Since it is important that undiluted probe solution does not touch the membrane, pour the ULTRAhyb-Oligo solution from the prehybridization into a 50 mL conical tube, add the labeled probe, mix by swirling, and then immediately pour the solution back into the container with the blot.
\item Hybridize overnight (6-24hr) at 42\degC with gentle agitation. It may be possible to reduce the hybridization time for detection of relatively abundant RNA, determined empirically.
\item Wash the blot 2x30 min at 42\degC. Immediately pour at least 50 mL stringency wash buffer onto the blot and incubate at 42\degC for 30 min with gentle agitation. Repeat with fresh stringency wash buffer.
\end{enumerate}

\section*{Detection}
\label{sec:detect}

%The assay described below detects the biotin-labeled full length and protected probes generated in the RPA assay and blotted onto a nylon membrane using the Ambion BrightStar BioDetect kit. The procedure is based on high affinity binding of alkaline phosphatase-conjugated streptavidin to biotin-labeled RNA, and detection of complexes by chemiluminescence using the CDP-Star substrate. The amount of biotin-labeled RNA can be quantified, because the amount of chemiluminescence is proportional to the amount of biotin contained in the RNA blotted onto the membrane.


\begin{enumerate}[resume]
  \item Measure the surface area of your membrane to determine the amounts of the washing and assay buffers needed. Dissolve any precipitate in the 5X washing buffer at 37-65\degC. Dilute the 5X washing buffer and 10X assay buffer to 1X working buffers before the experiment, prepare enough for one experiment only (discard the unused diluted buffer after each experiment). For example, for a 10 x 10 cm membrane (100 cm$^2$), 100 mL of washing buffer and 30 mL of assay buffer are needed (the volumes below are given for a 100 cm$^2$ membrane). Place the membrane sample side up in a plastic box of similar size. Do not allow the membrane to dry throughout the entire procedure. The procedure is performed at room temperature on a rocking shaker at medium speed (10-20 rockings/min). For abundant RNAs, it is possible to reduce the lengths of the (many) wash and block steps while still obtaining a clean blot with low background.
\item Wash the membrane 2 times, 5 min each, in 20 mL 1X washing buffer. 
\item Wash the membrane 2 times, 5 min each, in 10 mL 1X blocking buffer.
\item Wash the membrane once for 30 min in 20 mL 1X blocking buffer. 
\item Incubate the membrane for 30 min in 10 mL of the conjugate solution (10 mL
blocking buffer plus 1 \mul of streptavidin-alkaline phosphatase conjugate).   
\item Wash the membrane once for 15 min in 20 mL 1X blocking buffer. 
\item Wash the membrane 3 times, 15 min each, in 20 mL 1X washing buffer. 
\item Wash the membrane 2 times, 2 min each, in 15 mL 1X assay buffer. 
\item Incubate the membrane in enough CDP-star solution to cover, about 1mL, for 5 min. Rotate by hand to make sure that the solution evenly covers the entire membrane. 
\item Take out the membrane with forceps. Let the excess of CDP-star solution drip off, place the membrane on clear plastic wrap and wrap, ensuring no air bubbles are left.
%(or, if you can, in a transparent plastic bag, remove all the air, and heat-seal the bag. 
\item Image the membrane on the Chemi-doc or other digital imager. Use chemiluminescent detection with signal accumulation mode, for example 45 images, 900s.
The CDP-star reaches peak light emission in 2-4 h, and the light emission persists at the high level for several days, so you may get a better image by waiting minutes or hours to image. SYBR safe also tends to transfer to the membrane with rRNA, sometimes enough to be seen by the naked eye, but is visualizable on Cy3 channel; unfortunately loading dyes (bromophenol blue?) are also transferred, visible as a low-molecular weight bright blur on this channel so tRNA is not visible. 
\end{enumerate}

%\begin{enumerate}[resume]
%\item 
%\end{enumerate}

\section*{Stripping and re-probing}

Membranes may be stripped and re-probed multiple times (at least 4); they can be left in wash or block solution overnight, or frozen in plastic wrap at -20\degC.

\begin{enumerate}[resume]
\item Remove membrane from plastic wrap, place membrane in a plastic tray that can survive 100\degC liquids, covered by a minimal amount of NB strip buffer. Do not dry membrane, which may stick previous probes to membrane.
\item Twice: Heat 50mL NB strip buffer to boiling, in the microwave. Empty previous liquid, and pour boiling strip buffer over membrane in tray. Allow to cool to room temperature on rocker, about 15 minutes.
\item Restart prehybridization (step \ref{step:prehyb}) and continue as before.
\end{enumerate}


\end{document}